install/civicrm.php

```diff
@@ -36,6 +36,13 @@ function civicrm_setup($filesDirectory) {
   global $crmPath, $sqlPath, $pkgPath, $tplPath;
   global $compileDir;
 
+  // This is needed to allow CiviCRM to be installed by drush.
+  $pkgPath = $crmPath . DIRECTORY_SEPARATOR . 'packages';
+  set_include_path($crmPath . PATH_SEPARATOR .
+      $pkgPath . PATH_SEPARATOR .
+      get_include_path()
+  );
+
```

Comment:
It's usually preferable to call ```CRM_Core_ClassLoader::singleton()->register();``` to setup autoloading (for core code, for ```packages```, and for ```vendor```). Using the classloader makes it easier to reorganize the codebase (eg shifting classes from ```packages``` to ```vendor```/composer.json; or changing the relationship between the civicrm-core and civicrm-drupal repos).

FWIW, both ```install/index.php``` and ```civicrm.drush.inc``` (```_civicrm_init()``` - used for most commands, albeit perhaps not installation) call ClassLoader's ```register()```.